components/filemanager/preview-pane.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Preview Page
+page_title: FileManager Preview Page
+description: Preview Page in the FileManager for Blazor.
+slug: filemanager-preview-pane
+tags: telerik,blazor,filemanager,preview,pane
+published: True
+position: 9
+---
+
+# FileManager Preview Pane
+
+The Preview Pane shows additional information about the selected file or folder in the view.
+
+The preview pane visibility toggles by the Switch Tool in the FileManager Toolbar. By default, it shows the following information about the selected file or folder:
+
+* Thumbnail, representing the file
+* File extension
```

Comment:
Is it more of a file type category - we render "image" for images, rather than something like ".jpg".